common/djangoapps/student/roles.py

```diff
@@ -100,17 +145,29 @@ def get_roles(role):
         """
         return ACCESS_ROLES_INHERITANCE.get(role, set()) | {role}
 
+    # TODO: test that this works as expected
+    def get_all_roles_set(self):
+        return self._roles
```

Comment:
Please remove TODO and make this a `@property`.

Reply:
done